Wordle/Wordle/GameManager.cpp

```diff
@@ -0,0 +1,155 @@
+#include "GameManager.h"
+#include "FileIO.h"
+#include "DateTime.h"
+
+#include <iostream>
+#include <string>
+#include <cstdlib>
+#include <ctime>
+#include <vector>
+
+
+const unsigned sizeArray = 100;
+
+std::string getRandomWord()
+{
+    //get an array of words
+	std::string listWords[sizeArray];
+    readListWords("fiveLetterWords.txt", listWords, sizeArray);
+
+    //random
+    std::srand(static_cast<unsigned int>(std::time(nullptr)));
+
+    std::time_t currentTime = std::time(nullptr);
+
+    unsigned randomValue = std::rand() % sizeArray;
+
+    std::string randomWord = listWords[randomValue];
+    return randomWord;
+}
+
+bool compareWords(std::string randomWord)
```

Comment:
Better split into 2 functions. The one with actual two strings comparison so it's easy to find actual algorithm part and modify it if needed